### ISSUE TITLE
Segfault when entering a note in the Piano Roll Editor

### ISSUE THREAD

**Issue by theGreatWhiteShark** [hydrogen-music/hydrogen#1386]
Hydrogen version *          : 0bfb9b28befa5f08b83019c4cdd4978ece3a6dcd
Operating system + version  : 
Audio driver + version      : 

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Entering an arbitrary note in the PianoRollEditor makes my current Hydrogen version crash. This one looks quite severe.

**Comment by cme** [comment]
I don't think this should be on the 1.1.1 milestone? 1.1.1 will be off of `releases/1.1` and not off of `master`, and `releases/1.1` doesn't have this issue?

**Comment by theGreatWhiteShark** [comment]
> I don't think this should be on the 1.1.1 milestone? 1.1.1 will be off of releases/1.1 and not off of master, and releases/1.1 doesn't have this issue?

Ah. you are right. I probably clicked the wrong commit and got confused by the milestones it was present at.
